fuzz.go

```diff
@@ -263,7 +273,11 @@ func (fc *fuzzerContext) doFuzz(v reflect.Value, flags uint64) {
 	defer func() { fc.curDepth-- }()
 
 	if !v.CanSet() {
-		return
+		if fc.fuzzer.allowUnexportedFields && v.CanAddr() {
```

Comment:
Sorry I was too slow making this comment -- can you flip the logic around on this?

```
if !fc.fuzzer.allowUnexportedFields || !v.CanAddr() {
  return
}
v = reflect.NewAt(v.Type(), unsafe.Pointer(v.UnsafeAddr())).Elem()
```

This reduces the indentation. Minor style nit.

Reply:
No problem! I just did that